appcues/src/main/java/com/appcues/DeepLinkHandler.kt

```diff
@@ -39,7 +41,16 @@ internal class DeepLinkHandler(
         return when {
             segments.count() == 2 && segments[0] == "experience_preview" -> {
                 appcuesCoroutineScope.launch {
-                    experienceRenderer.preview(segments[1])
+                    experienceRenderer.preview(segments[1])?.let {
+                        val frameId = if (it.experience.renderContext is RenderContext.Embed)
+                            it.experience.renderContext.frameId else String()
+
+                        Toast.makeText(
```

Comment:
should this be a snackbar? we cant re-use the same ones we have in debugger.

Reply:
I think since Android has the native Toast it makes sense to just use it for this utility case, as oppose to iOS where the debugger item was ported over. Porting the debugger item over would prob be fine too, but just extra work. In terms of the end user UX toast/snackbar, maybe best to run that by Sam.